core/src/test/scala/kanaloa/reactive/dispatcher/queue/WorkerIdleSpec.scala

```diff
@@ -2,52 +2,40 @@ package kanaloa.reactive.dispatcher.queue
 
 import akka.actor.PoisonPill
 import kanaloa.reactive.dispatcher.queue.Queue.{NoWorkLeft, Unregister}
-import kanaloa.reactive.dispatcher.queue.Worker.Hold
+import kanaloa.reactive.dispatcher.queue.Worker.SetDelay
 
 import scala.concurrent.duration._
 
 class WorkerIdleSpec extends WorkerSpec {
 
   "An Idle Worker" should {
-    "status is Idle" in withIdleWorker { (worker, queueProbe, routeeProbe) ⇒
+    "status is Idle" in withIdleWorker() { (worker, _, _, _) ⇒
       assertWorkerStatus(worker, Worker.Idle)
     }
 
-    "terminate if NoWorkLeft" in withIdleWorker { (worker, queueProbe, routeeProbe) ⇒
+    "terminate if NoWorkLeft" in withIdleWorker() { (worker, _, _, _) ⇒
       worker ! NoWorkLeft
       expectTerminated(worker)
     }
 
-    "terminate if Queue terminates" in withIdleWorker { (worker, queueProbe, routeeProbe) ⇒
+    "terminate if Queue terminates" in withIdleWorker() { (worker, queueProbe, _, _) ⇒
       queueProbe.ref ! PoisonPill
       expectTerminated(worker)
     }
 
-    //TODO: assert that the value was properly applied, not just set to None
-    "consume Hold value when sent Work" in withIdleWorker { (worker, queueProbe, routeeProbe) ⇒
-      worker ! Hold(1.millisecond)
-      eventually {
-        worker.underlyingActor.delayBeforeNextWork.value shouldBe 1.millisecond
-      }
+    "send work to Routee and becomes Working" in withIdleWorker() { (worker, _, routeeProbe, _) ⇒
       worker ! Work("work")
       routeeProbe.expectMsg("work")
-      worker.underlyingActor.delayBeforeNextWork shouldBe None
       assertWorkerStatus(worker, Worker.Working)
     }
 
-    "send work to Routee and becomes Working" in withIdleWorker { (worker, queueProbe, routeeProbe) ⇒
```

Comment:
this test wasn't removed, it was the one above, which is no longer valid, since now idle worker won't get a Hold from external actors